docs/_docs/persistence/snapshots.adoc

```diff
@@ -160,27 +166,27 @@ the topology and wait while the data gets rebalanced and indexes are rebuilt.
 == Consistency Guarantees
 
 All snapshots are fully consistent in terms of concurrent cluster-wide operations as well as ongoing changes with Ignite
-Persistence and other files on nodes.
+Persistence and files on nodes.
```

Comment:
It's unclear what are those other "files".  In my version, I wanted to say that all the changes with Ignite Persistence and other files are to be consistently recorded to a snapshot. The "other files" imply some other files that unrelated to persistence and it's not that important to mention here.

In the current version, we have just "files" (without "other"). And we either need to return "other" or be more specific listing the names of those files.

Reply:
Fixed.